codes/quantum/qubits/15_qubit_code.yml

```diff
@@ -0,0 +1,91 @@
+#######################################################
```

Comment:
Please check out https://arxiv.org/abs/1612.07330 and add the aspects of this code that they discuss: 
- connection to magic state distillation
- color code cousin connection, with "detail" statement

Reply:
Also add discussion of recent paper https://arxiv.org/pdf/2112.01446.pdf.  It shows how to "morph" the 15 qubit code into a 10 qubit code with a fault-tolerant T-gate.

Reply:
Done.